lib/cfn-nag/cli_options.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require 'trollop'
+
+# rubocop:disable Metrics/BlockLength
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+def cli_options(cfn_nag_scan: false)
+  options_message = '[options] <cloudformation template path ...>|' \
```

Comment:
nit: move the messages into the "scope" where they are relevant and have the shared stuff up here only